### ISSUE TITLE
discuss: tutoring people on the command line

### ISSUE THREAD

**Issue by coolaj86** [dashhive/crowdnode-cli#15]
> Maybe add a step how to open up a terminal for people who don't know what they're doing.

One of the beauties of the command line is that it's great for automating and scripting things that you do often.

Teaching people how who don't know how to open a Terminal how to do everything from scratch for the first time I think falls outside the scope of this project.

If they're on Windows, they _could_ use cmd.exe, but that will likely not work. You _could_ ask them to download Microsoft Terminal from the Windows Store, but then you have to teach them the differences between cmd and PowerShell. You could ask them to https://webinstall.dev/wsl (which is why I created it - to help people learn to program in a normal environment without having to learn all of the ins and outs of Windows first), but...

It's a rabbit hole.

If you have an idea of how to solve this, feel free. I don't feel like I'm the right person to take that on because my mind gets all jumbled up around first principles, I get analysis paralysis and a year later I still don't have so much as a "how to get started as a developer" mini course.

My thought on this wasn't to try to convert people to the command line, but rather to make an excellent tool for developers and power users that they can inspect and see how the pieces come together, and built scripts to do what they want.

Granted, the current CLI code fails at that because it became too complicated, but the SDK hits the spot, I believe.

**Comment by riongull** [comment]
You're right.  A better approach would be to supplement the CLI with a blog post and link to it in the README.  I can task that when we're ready (after a few more reviews from other devs who try the tool).
